typha/fv-tests/state_recorder.go

```diff
@@ -112,3 +147,22 @@ func (r *StateRecorder) BlockAfterNUpdates(n int, duration time.Duration) {
 	r.blockAfter = n
 	r.blockDuration = duration
 }
+
+func (r *StateRecorder) KVCompareFn(kvs map[string]api.Update) func() error {
```

Comment:
This is an optimisation to make tests with lots of KVs workable (was using it for playtime very large scale testing).